lib/rspecq/worker.rb

```diff
@@ -15,16 +15,17 @@ class Worker
     # will be split and scheduled on a per-example basis.
     attr_accessor :file_split_threshold
 
-    attr_reader :queue
+    attr_reader :queue, :max_requeues
 
-    def initialize(build_id:, worker_id:, redis_host:, files_or_dirs_to_run:)
+    def initialize(build_id:, worker_id:, redis_host:, files_or_dirs_to_run:, max_requeues:)
```

Comment:
Since it's not strictly required (it'll default to `3` anyway), I'd prefer to make this a plain attribute instead of an argument, just like `populate_timings`. This way we keep the `Worker` API more convenient, by using sensible defaults.

(Side: In the same vein, we should also make `files_or_dirs_to_run` a plain instance attribute instead of an argument, but that's out of scope of this PR)